Source/CarthageKit/Xcode.swift

```diff
@@ -1180,6 +1240,8 @@ public func buildInDirectory(directoryURL: NSURL, withOptions options: BuildOpti
 		let locator = buildableSchemesInDirectory(directoryURL, withConfiguration: options.configuration, forPlatforms: options.platforms)
 			.replayLazily(Int.max)
 
+		var urls: [NSURL] = []
```

Comment:
It'd be preferable to not use local state to pass information outside of the signal chain.